actions/helm-deploy/action.yaml

```diff
@@ -0,0 +1,92 @@
+name: "Deploy helm chart"
+description: "Deploy a helm chart to the GKE cluster"
+inputs:
+  gke-service-account:
+    description: "GKE service account key for authentication"
+    required: true
+  gke-project:
+    description: "GKE project id for authentication"
+    required: true
+  gke-region:
+    description: "GKE region for authentication"
+    required: true
+  gke-cluster:
+    description: "GKE cluster for authentication"
+    required: true
+  release-name:
+    description: "Helm release name"
+    required: true
+  namespace:
+    description: "K8s namespace to deploy in"
+    required: true
+  chart:
+    description: "Helm chart to deploy"
+    required: true
+  chart-version:
+    description: "Chart Version"
+    required: true
+  values-yaml:
+    description: "Location of helm's value file"
+    required: true
+  repository-name:
+    description: "Helm repository name"
+    required: false
+  repository-url:
+    description: "Url of the repository"
+    required: false
+  python-version:
+    description: "Python version"
+    default: "3.10"
+    required: false
+  gcloud-sdk-version:
+    description: "GCloud SDK version"
+    default: "376.0.0"
+    required: false
+  kubectl-version:
+    description: "Kubectl version"
+    default: "v1.23.0"
+    required: false
+  helm-version:
+    description: "Helm version"
+    default: "v3.8.1"
+    required: false 
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Setup environment
+      uses: bakdata/ci-templates/actions/helm-setup@feature/helm-deploy-destroy-workflows
+      with:
+        gke-service-account: ${{ inputs.gke-service-account }}
+        gke-project: ${{ inputs.gke-project }}
+        gke-region: ${{ inputs.gke-region }}
+        gke-cluster: ${{ inputs.gke-cluster }}
+        python-version: ${{ inputs.python-version }}
+        gcloud-sdk-version: ${{ inputs.gcloud-sdk-version }}
+        kubectl-version: ${{ inputs.kubectl-version }}
+        helm-version: ${{ inputs.helm-version }}
+
+    - name: Add helm repository
+      shell: bash
+      run: |
+        if [ -n "$REPOSITORY_NAME" ] && [ -n "$REPOSITORY_URL" ]
+        then
+          helm repo add "$REPOSITORY_NAME" "$REPOSITORY_URL"
+          helm repo update
+        else
+          echo "No repository specified; Skipping step"
+        fi
+      env:
+        REPOSITORY_NAME: ${{ inputs.repository-name }}
+        REPOSITORY_URL: ${{ inputs.repository-url }}
+
+    - name: Deploy helm chart
+      shell: bash
+      run: |
+        helm upgrade --debug --install --wait --timeout 1200s --force \
+          --version ${{ inputs.chart-version }} \
+          --values ${{ inputs.values-yaml }} \
```

Comment:
What if we have multiple values? Can we use an array parameter?

Reply:
Now the chart allows a string for a single helm values file (like before) or additionally a json array like this: `value-files: ["foo/values.yaml", "foo/values-1.yaml"]` for multiple value files